transformer_engine/pytorch/ops/basic/layer_norm.py

```diff
@@ -184,17 +185,21 @@ def op_forward(
     ) -> torch.Tensor:
 
         # Check tensor dims
+        weight = self.weight
+        weight_dims = tuple(weight.size())
         input_dims = tuple(input_.size())
```

Comment:
apparently `torch.Size` is a subclass of `tuple` so tuple creation probably not needed